vng_api_common/models.py

```diff
@@ -69,106 +61,3 @@ class Meta:
 
     def __str__(self):
         return self.identifier
-
-
-class APICredential(models.Model):
```

Comment:
There should be some warning in the CHANGELOG in the future, that the model to change credentials has been changed.